chall02/fulldemo.py

```diff
@@ -0,0 +1,27 @@
+#! /usr/local/bin/python3
+
+import sys
+num = len(sys.argv)
+if num != 2: 
+	print("usage: ./fulldemo.py <a-zA-Z string>")
```

Comment:
You should make your error message more dynamic, what if we come to change the file's name?
```suggestion
	print("usage: %s <a-zA-Z string>" % sys.argv[0])
```